hail_scripts/reference_data/clinvar.py

```diff
@@ -156,8 +157,7 @@ def _parse_clinvar_release_date(local_vcf_path: str) -> str:
     with gzip.open(local_vcf_path, 'rt') as f:
         for line in f:
             if line.startswith('##fileDate='):
-                clinvar_release_date = line.split('=')[-1].strip()
```

Comment:
New version of `ruff` is more aggressive about these, which I think is a good think!